### DISCUSSION TITLE
How to Eliminate Bonding Errors Caused by Aromaticity？

### DISCUSSION THREAD

**Discussion by ShangChien** [epam/ketcher#1249] [Ideas]
When benzene is added to the benzene ring, can the valence error be automatically eliminated by the aromaticity？
![182223](https://user-images.githubusercontent.com/48350636/153575201-42db4957-6548-45e2-abfe-0957fda6ae04.png)
it does not work when the aroma button is clicked, any suggestion?

**Comment by AndreiMazol** [comment]
Dear @ShangChien ,

this issue will be fixed in scope of the nearest releases, issue https://github.com/epam/ketcher/issues/682

**Reply by ShangChien** [reply]
Thank you for your team work, ketcher is useful tool with large potential in web components.
